hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/management/TargetAssignmentOperations.java

```diff
@@ -105,7 +105,7 @@ public static void saveAllAssignments(final List<Target> targets, final List<Dis
         final String maintenanceTimeZone = maintenanceWindowLayout.getMaintenanceTimeZone();
 
         final Set<Long> dsIds = distributionSets.stream().map(DistributionSet::getId).collect(Collectors.toSet());
-        final List<DeploymentRequest> deploymentRequests = new ArrayList<>();
+               final List<DeploymentRequest> deploymentRequests = new ArrayList<>();
```

Comment:
accidentally wrong indentation

Reply:
I can see this indented correctly in my eclipse, not sure how this is still showing a difference :(